wpilibc/src/main/native/cpp/CameraServer.cpp

```diff
@@ -177,64 +177,7 @@ static std::string PixelFormatToString(int pixelFormat) {
       return "Unknown";
   }
 }
-#if 0
```

Comment:
@PeterJohnson I assume you plan to come back to this?

Reply:
Probably, but I'm okay with removing it for now.  That's what version history is for.